project_resource_calendar/static/src/css/print_report_header_style.css

```diff
@@ -0,0 +1,14 @@
+.h1-print_report_header {
```

Comment:
These are all the same, why put it on different classes? Are they going to change? if they change we can slip, but right now they are the same. Since you already split, it;s ok... Missing new line at the end

Reply:
The name of the file, says header, but you are defining body and footer. Better name: one_time_booking_report.xml

Reply:
@eilst  Add a new line Done